resources/views/checkout/pages/credentials.blade.php

```diff
@@ -6,11 +6,17 @@
 
 @section('content')
     <div class="container">
-        <x-rapidez-ct::layout class="mt-4 sm:mt-12">
+        <x-rapidez-ct::layout class="lg:items-end">
             <x-rapidez-ct::title-progress-bar :href="route('cart')" :$checkoutSteps :$currentStep :$currentStepKey>
```

Comment:
i see a duplicate of the exact same code, maybe create a component for it?

Reply:
I tried to create a component for this locally but it didn't really make it cleaner,
I think we should keep it this way